models/turbine_models/custom_models/llm_runner.py

```diff
@@ -183,14 +181,14 @@ def run_llm(
         streaming_llm=streaming_llm,
     )
     if not chat_mode:
+        prompt = append_user_prompt(chat_sys_prompt, prompt)
         initial_input = tokenizer(prompt, return_tensors="pt")
         example_input_id = initial_input.input_ids
         turbine_results = llm.generate(example_input_id)
         return tokenizer.decode(turbine_results)
-    prompt = chat_sys_prompt
     while True:
         user_prompt = input("User prompt: ")
-        prompt = append_user_prompt(prompt, user_prompt)
+        prompt = append_user_prompt(chat_sys_prompt, user_prompt)
```

Comment:
This is actually no bueno for chat because doing this you lose track of your chat history/context.